source/js/front/form.js

```diff
@@ -1,91 +1,130 @@
-                const removeButton = repeater.querySelector('.btn-repeater-remove');
-                addButton.addEventListener('click', (e) => {
-                    const subFieldsClone = repeater.querySelector('.sub-fields').cloneNode(true);
-                    subFieldsClone.querySelectorAll('input').forEach(field => {field.value = '';});
-                    repeater.insertBefore(subFieldsClone, addButton);
-                    if (repeater.querySelectorAll('.sub-fields').length > 1) {
-                        removeButton.classList.remove('u-display--none');
-                    }
-                });
-                removeButton.addEventListener('click', (e) => {
-                    const subFields =  repeater.querySelectorAll('.sub-fields');
-                    if (subFields.length > 1) {
-                        subFields[subFields.length - 1].remove();
-                    }
-                    if (subFields.length === 1) {
-                        removeButton.classList.add('u-display--none');
-                    }
-                });
+            return false;
+        });
+
+        if (conditionResult.every((x) => x === true)) {
+            field.classList.remove('u-display--none');
+            field.querySelectorAll('input').forEach((x) => {
+                x.required = true;
+            });
+        } else {
+            field.classList.add('u-display--none');
+            field.querySelectorAll('input').forEach((x) => {
+                x.required = false;
             });
         }
-    }
-;
-
+    },
+    setupRepeaters: (form) => {
+        const repeaters = form.querySelectorAll('.js-repeater');
+        repeaters.forEach((repeater) => {
+            const addButton = repeater.querySelector('.btn-repeater-add');
+            const removeButton = repeater.querySelector('.btn-repeater-remove');
+            addButton.addEventListener('click', (e) => {
+                const subFieldsClone = repeater
+                    .querySelector('.sub-fields')
+                    .cloneNode(true);
+                subFieldsClone.querySelectorAll('input').forEach((field) => {
+                    field.value = '';
+                });
+                repeater.insertBefore(subFieldsClone, addButton);
+                if (repeater.querySelectorAll('.sub-fields').length > 1) {
+                    removeButton.classList.remove('u-display--none');
+                }
+            });
+            removeButton.addEventListener('click', (e) => {
+                const subFields = repeater.querySelectorAll('.sub-fields');
+                if (subFields.length > 1) {
+                    subFields[subFields.length - 1].remove();
+                }
+                if (subFields.length === 1) {
+                    removeButton.classList.add('u-display--none');
+                }
+            });
+        });
+    },
+    setupRemoteSelect: (form) => {
+        var apiUrl = eventintegration.apiurl;
```

Comment:
- 'var' instead of 'let' typo? 
- add check eg. 'eventintegration?.apiurl' and bailout if its undefined